### PR TITLE
feat(telephony): remove the mgcp banner

### PR DESCRIPTION
**Pull request by antonymarion**
ref: MANAGER-12730

| Question         | Answer
| ---------------- | ---
| Branch?          | `master` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Feat #MANAGER-12730
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [ ] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)

## Description

Use feature flipping to remove temporarly mgcp banner

### CODE REVIEW

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/line-main.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
         <oui-message
             data-type="warning"
-            data-ng-if="LineCtrl.displayMgcpBanner"
+            data-ng-if="LineCtrl.isMgcpBannerAvailable() && LineCtrl.displayMgcpBanner"
```
Comment (darsene) [inline packages/manager/apps/telecom/src/app/telecom/telephony/line/line-main.html:24]: As conditions are linked, you should set `displayMgcpBanner` only if `isMgcpBannerAvailable()` is `true`

Comment (antonymarion) [reply packages/manager/apps/telecom/src/app/telecom/telephony/line/line-main.html:24]: Indeed

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/line.controller.js [hunk 2]
```diff
@@ -103,8 +105,10 @@ export default /* @ngInject */ function TelecomTelephonyLineCtrl(
 
         promises.push(
           self.line.getPhone().then(() => {
-            self.displayMgcpBanner =
-              self.line.phone.protocol === PHONE_PROTOCOL.MGCP;
+            if (this.isMgcpBannerAvailable()) {
+              self.displayMgcpBanner =
+                self.line.phone.protocol === PHONE_PROTOCOL.MGCP;
+            }
```
Comment (qpavy) [inline packages/manager/apps/telecom/src/app/telecom/telephony/line/line.controller.js:111]:
```suggestion
              self.displayMgcpBanner = this.isMgcpBannerAvailable() && self.line.phone.protocol === PHONE_PROTOCOL.MGCP;
```

Comment (antonymarion) [reply packages/manager/apps/telecom/src/app/telecom/telephony/line/line.controller.js]: Nice!

**Review by qpavy — APPROVED** [review]

**Review by dectotam — APPROVED** [review]

**Review by JacquesLarique — APPROVED** [review]

**Pull request merged**